docs/source/user-guide/plugins/your-first-polars-plugin.md

```diff
@@ -1,288 +0,0 @@
-# Your first Polars plugin
```

Comment:
Just checking - you don't want to keep this title?

Reply:
Personally, I think it is a bit misleading as you are not developing your first plugin. It refers to another tutorial at the end to create an actual usable plugin.